gcp/website/frontend_emulator.py

```diff
@@ -0,0 +1,358 @@
+          "CVE-2023-4194", "CVE-2023-4244", "CVE-2023-4273", "CVE-2023-42753",
+          "CVE-2023-42755", "CVE-2023-42756", "CVE-2023-4622", "CVE-2023-4623",
+          "CVE-2023-4921"
+      ],
+      source='test',
+      public=True,
+      import_last_modified=datetime.datetime(2024, 1, 9),
+  ).put()
+
+  osv.Bug(
+      id='SUSE-SU-2023:3313-1',
+      db_id='SUSE-SU-2023:3313-1',
+      status=1,
+      upstream_raw=[
+          "CVE-2022-40982", "CVE-2023-0459", "CVE-2023-20569", "CVE-2023-21400",
+          "CVE-2023-2156", "CVE-2023-2166", "CVE-2023-31083", "CVE-2023-3268",
+          "CVE-2023-3567", "CVE-2023-3609", "CVE-2023-3611", "CVE-2023-3776",
+          "CVE-2023-4004"
+      ],
+      source='test',
+      public=True,
+      import_last_modified=datetime.datetime(2023, 8, 14),
+      timestamp=datetime.datetime(2023, 8, 14),
+  ).put()
+
+  osv.UpstreamGroup(
+      id='SUSE-SU-2023:3313-1',
+      db_id='SUSE-SU-2023:3313-1',
+      upstream_ids=[
+          "CVE-2022-40982", "CVE-2023-0459", "CVE-2023-20569", "CVE-2023-21400",
+          "CVE-2023-2156", "CVE-2023-2166", "CVE-2023-31083", "CVE-2023-3268",
+          "CVE-2023-3567", "CVE-2023-3609", "CVE-2023-3611", "CVE-2023-3776",
+          "CVE-2023-4004"
+      ],
+      last_modified=datetime.datetime(2023, 8, 14)).put()
+
+  osv.Bug(
+      id="CVE-2021-44228",
+      db_id="CVE-2021-44228",
+      public=True,
+      last_modified=datetime.datetime(2025, 2, 4),
+      source="test",
+      timestamp=datetime.datetime(2023, 8, 14),
+  ).put()
+
+  osv.Bug(
+      id="CYCLE-ROOT-1",
+      db_id="CYCLE-ROOT-1",
+      upstream_raw=['CYCLE-ROOT-2'],
+      public=True,
+      last_modified=datetime.datetime(2025, 2, 4),
+      source="test",
+      timestamp=datetime.datetime(2023, 8, 14)).put()
+  osv.Bug(
+      id="CYCLE-ROOT-2",
+      db_id="CYCLE-ROOT-2",
+      upstream_raw=['CYCLE-ROOT-1'],
+      public=True,
+      last_modified=datetime.datetime(2025, 2, 4),
+      source="test",
+      timestamp=datetime.datetime(2023, 8, 14)).put()
+  osv.Bug(
+      id="CYCLE-ROOT-3",
+      db_id="CYCLE-ROOT-3",
+      upstream_raw=['CYCLE-ROOT-1', 'CYCLE-ROOT-2'],
+      public=True,
+      last_modified=datetime.datetime(2025, 2, 4),
+      source="test",
+      timestamp=datetime.datetime(2023, 8, 14)).put()
+
+  osv.UpstreamGroup(
+      id='CYCLE-ROOT-3',
+      db_id='CYCLE-ROOT-3',
+      upstream_ids=['CYCLE-ROOT-1', 'CYCLE-ROOT-2'],
+      last_modified=datetime.datetime(2023, 8, 14)).put()
+  osv.UpstreamGroup(
+      id='CYCLE-ROOT-1',
+      db_id='CYCLE-ROOT-1',
+      upstream_ids=['CYCLE-ROOT-2'],
+      last_modified=datetime.datetime(2023, 8, 14)).put()
+
+  osv.UpstreamGroup(
+      id='CYCLE-ROOT-2',
+      db_id='CYCLE-ROOT-2',
+      upstream_ids=['CYCLE-ROOT-1'],
+      last_modified=datetime.datetime(2023, 8, 14)).put()
+
+
+if __name__ == '__main__':
+  ds_emulator = tests.start_datastore_emulator()
+  import main
```

Comment:
Maybe add a comment to explain we need to start the emulator before importing main to make the global ndb client use the emulator.